### PR TITLE
Issue #33 toggle stuck on dark & Issue #16 styling toggle button

### LINKED ISSUE #16: Style - Move and Style Light/Dark Theme Toggle

### LINKED ISSUE #33: Bug - Light/Dark Toggle Currently Stuck on Dark Mode

### PR DESCRIPTION
**Pull request by audreyclarkdev**
I've made a lot of changes here

- Was able to fix dark and light mode working correctly with the toggle
- Rearranged some of the order of the css 
- Styled the toggle and moon  #icon
- Added dictionary favicon images and updated favicon to point to them
- Capitalized "Dictionary Web App" title
- Made the text bold inside the search input bar

#33 & #16

### CODE REVIEW

File: start-here/styles/style.css [hunk 1]
```diff
@@ -223,6 +196,92 @@ label[for="mono"] {
...
+
+/* After dark mode is enabled - toggle on */
+.toggle-slider::after {
+  content: "";
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  top: 0px; 
+  left: 0px;
+  background-color: #ccc;
```
Comment (knopgm) [inline start-here/styles/style.css]: Change these values to get a little padding to the toggle button.  I'm not sure about how much but in this image below I was pretty close:
image

**Review by knopgm — CHANGES_REQUESTED** [review]
Add a little gap between the toggle button and the button wrapper.

**Comment by audreyclarkdev** [comment]
@knopgm and @NuraSad 
I made some changes, and I think it looks better now. :) Thanks for the comment Gabriela!

**Comment by knopgm** [comment]
Great job!

**Review by knopgm — APPROVED** [review]

**Review by lexoskeleton — APPROVED** [review]
This looks great!

**Pull request merged**